microstream-cache/src/main/java/io/micronaut/microstream/cache/CacheConfigurationProperties.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.microstream.cache;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+
+/**
+ * Configuration for Microstream Cache module.
+ * @author Sergio del Amo
+ * @since 1.0.0
+ */
+@ConfigurationProperties(MicrostreamCacheConfigurationProperties.PREFIX)
+public class CacheConfigurationProperties implements CacheConfiguration {
```

Comment:
I'm also unclear about the usefulness of this class. Can't its content simply be part of `MicrostreamCacheConfigurationProperties`?

Reply:
Looking closer at the code it seems it isn't even used anywhere else. Is it maybe mistakenly left over?

Reply:
We can delete `io/micronaut/microstream/cache/CacheConfiguration` but we will have to barcode the configuration in the docs.

Reply:
<img width="1213" alt="Screenshot 2022-05-09 at 22 06 48" src="https://user-images.githubusercontent.com/864788/167488975-a48d7982-602c-4e63-9e77-db3a464bac7a.png">